Makefile

```diff
@@ -34,6 +35,10 @@ all-srcs:
 .PHONY: fmt-vet-lint-test
 fmt-vet-lint-test: fmt vet lint test
 
+.PHONY: perf-test
+perf-test: otelsvc
+	$(MAKE) -C testbed runtests
```

Comment:
While we shouldn't run the testbed every time it should be built together with the repo.

Reply:
Fine to do it later.